Find a functionally equivalent code:android.content.RestrictionEntry.getSelectedString:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the currently selected string value.
 * @return the currently selected value, which can be null for types that aren't for holding
 * single string values.
 */

@@ -1,3 +1,3 @@
 {
-    return currentValue;
+    return mCurrentValue;
 }

